go/chat/sender.go

```diff
@@ -817,6 +817,19 @@ func (s *BlockingSender) Send(ctx context.Context, convID chat1.ConversationID,
 					clearedCache = true
 				}
 				continue
+			case libkb.ChatEphemeralRetentionPolicyViolatedError:
+				s.Debug(ctx, "Send: failed because of invalid ephemeral policy, trying the whole thing again")
+				if !clearedCache {
+					s.Debug(ctx, "Send: clearing inbox cache to retry getting ephemeral policy")
+					s.G().InboxSource.Clear(ctx, sender)
```

Comment:
This is a bold move, why not just remote only the GetUnverifiedConv call?

Reply:
yeah i like that better